portfolio/src/main/webapp/script.js

```diff
@@ -33,8 +33,17 @@ function addRandomGreeting() {
 
 async function addMotivationalMessage() {
   const responseFromServer = await fetch('/hello');
-  const textFromResponse = await responseFromServer.text();
+  const textFromResponse = await responseFromServer.json();
+    
+  const messageContainer = document.getElementById('message-container'); 
+  messageContainer.innerHTML = '';
+  
+  messageContainer.appendChild(createListElement(textFromResponse[0]));
+  
```

Comment:
This line does not look formatted correctly. Make should the indentation is correct and you can remove an extra blank line above

Reply:
fixed